frontend/src/views/auth/Register.vue

```diff
@@ -5,49 +5,48 @@
       <e-text-field
         v-model="firstname"
         :name="$tc('entity.user.fields.firstname')"
+        vee-rules="required"
         append-icon="mdi-account-outline"
         dense
-        required
```

Comment:
This breaks the red-star-styling that was introduced by @pmattmann in https://github.com/ecamp/ecamp3/pull/2845

Reply:
That is the reason why I am setting the required attribute on ValidationProvider. ValidationProvider does not do anything with it, but the CSS relies on a parent element having that attribute.

Reply:
More generally, I'd like to propose that we don't use the `required` attribute directly, but rather keep all validation rules in one single property `vee-rules`. If we use `required`, we will at some point start using `pattern` and other attributes as well, and it's going to be a mess. `required` is an attribute on the native HTML `<input>` element, we don't have to pretend our `<e-text-field>` is an input, because it is not.